instance/models/openedx_instance.py

```diff
@@ -111,10 +115,7 @@ def domain(self):
 
         Returns external domain if present, otherwise falls back to internal domain.
         """
-        if self.external_lms_domain:
-            return self.external_lms_domain
-        else:
-            return self.internal_lms_domain
+        return self.get_domain('lms')
```

Comment:
@smarnach It buys us protection from typos. :) Obviously, my suggestion wasn't to address the problem you highlighted, because this comment was posted before you highlighted it :)

Also, you might misunderstood what I meant - I only suggested to extract those magic strings "lms", "studio", "ecommerce" etc. into constants - that's it.